diff --git a/lib/devices/android/bootstrap/src/io/appium/android/bootstrap/SocketServer.java b/lib/devices/android/bootstrap/src/io/appium/android/bootstrap/SocketServer.java
index b43aefe..8938361 100644
--- a/lib/devices/android/bootstrap/src/io/appium/android/bootstrap/SocketServer.java
+++ b/lib/devices/android/bootstrap/src/io/appium/android/bootstrap/SocketServer.java
@@ -117,7 +117,7 @@ class SocketServer {
       @Override
       public void run() {
         try {
-          watchers.check();
+          //watchers.check();
         } catch (final Exception e) {
         }
       }
@@ -143,8 +143,8 @@ class SocketServer {
 
   public void dismissCrashAlerts() {
     try {
-      new UiWatchers().registerAnrAndCrashWatchers();
-      Logger.debug("Registered crash watchers.");
+      //new UiWatchers().registerAnrAndCrashWatchers();
+      Logger.debug("Registered crash watchers. NO");
     } catch (Exception e) {
       Logger.debug("Unable to register crash watchers.");
     }
